packages/synapse-react-client/src/components/CardMetadataDisplay/CardMetadataDisplay.tsx

```diff
@@ -0,0 +1,159 @@
+import { QueryBundleRequest } from '@sage-bionetworks/synapse-types'
+import {
+  getAdditionalFilters,
+  parseEntityIdFromSqlStatement,
+  SQLOperator,
+} from '../../utils/functions'
+import {
+  BUNDLE_MASK_QUERY_RESULTS,
+  BUNDLE_MASK_QUERY_SELECT_COLUMNS,
+} from '../../utils/SynapseConstants'
+import useGetQueryResultBundle from '../../synapse-queries/entity/useGetQueryResultBundle'
+import { QueryVisualizationWrapperProps } from '../QueryVisualizationWrapper'
+import { SynapseCardLabel } from '../GenericCard'
+import { LabelLinkConfig } from '../CardContainerLogic'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  Typography,
+} from '@mui/material'
+import { getFieldIndex } from '../../utils/functions/queryUtils'
+
+export type CardMetadataDisplayProps = {
+  sql: string
+  sqlOperator?: SQLOperator
+  searchParams?: Record<string, string>
+  columnLinks?: LabelLinkConfig
+  labels?: string[]
+  columnAliases?: Record<string, string>
```

Comment:
Not sure how I feel about `labels` as a prop name but I like that it's succinct--I think that it would be more clear if you added a docstring for every prop. It would help understand what's happening, especially now that external folks are increasingly developing in this package

```suggestion
  /** The column names referring to the ordered keys in the key-value pairs that should be shown */
  labels?: string[]
  /** Optional display value overrides for column names */
  columnAliases?: Record<string, string>
```